### PR TITLE
CreateToken to composition API

### PR DESCRIPTION
**Pull request by leo-anderson-x**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Related wtih #4750 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=12RA5J7wGg6bDWPTZG359Pn5bCzXSJRfsifo3UktqVAjhJ9k&usdamount=0&donation=true)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 2ec3839</samp>

Refactor `CreateToken` component to use Composition API and improve prop naming for clarity.

### <samp>🤖 Generated by Copilot at 2ec3839</samp>

> _The `CreateToken` component got a new look_
> _It now uses the Composition API for every hook_
> _The `Loader` and `SubmitButton` props were renamed_
> _To avoid confusion and make them more clear_
> _This pull request deserves a thumbs up or a cheer_

### CODE REVIEW

File: components/bsx/Create/CreateToken.vue [hunk 1]
```diff
@@ -66,258 +69,255 @@
...
+const loadCollectionMeta = async () => {
+  const metadata = collections.value.map(({ metadata }) => metadata)
+
+  metadata.forEach(async (m, i) => {
+    try {
+      const meta = await fetchCollectionMetadata(collections[i])
+      set(collections.value, i, {
+        ...collections.value[i],
+        ...meta,
+      })
```
Comment (roiLeo) [inline components/bsx/Create/CreateToken.vue:197]: use spread operator or make it reactive

**Comment by roiLeo** [comment]
Tested with `/snek/gallery/3124270296-11`

**Comment by vikiival** [comment]
Cc @prury 

@leo-anderson-x have you tested that? 🥹

**Comment by leo-anderson-x** [comment]
> @leo-anderson-x have you tested that? 🥹

@vikiival I've only tested it briefly on snek and it would be nice to have @prury's verification!

File: components/bsx/Create/CreateToken.vue (hunk 1 above)
Comment (leo-anderson-x) [reply components/bsx/Create/CreateToken.vue]: I gave reactivity to the meta properties by setting their initial value to `collections.value`

**Comment by prury** [comment]
testing now

**Comment by prury** [comment]
RMRK1 test: 
https://kusama.subscan.io/tx/0x0680cf8826e0bbc283d052d6eaa06bf921c957b3c24cd62359d40c5808ea0aa4 ❌

**Comment by prury** [comment]
> > RMRK1 test: https://kusama.subscan.io/tx/0x0680cf8826e0bbc283d052d6eaa06bf921c957b3c24cd62359d40c5808ea0aa4 ❌
> 
> The changes only impact bsx & snek chain

my bad, running bsx/snek tests now

**Comment by prury** [comment]
ok, it works ✔️ 
https://deploy-preview-6896--koda-canary.netlify.app/bsx/gallery/5347553-2
https://basilisk.subscan.io/tx/0x0f2e61715b93c2343af5b676315f45bbfa9b6b64df2a149898dc5bfd2ccd6f6a
took a while because i found two unrelated bugs in the process

**Comment by leo-anderson-x** [comment]
@vikiival Waiting for your confirmation 😃

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $70 @ 4.47 USD/DOT ~ 15.66 $DOT
🧗 [12RA5J7wGg6bDWPTZG359Pn5bCzXSJRfsifo3UktqVAjhJ9k](https://polkadot.subscan.io/account/12RA5J7wGg6bDWPTZG359Pn5bCzXSJRfsifo3UktqVAjhJ9k)
🔗 [0xe640a2af11b057a60c509a116c2ffe372c4524493377757c4bbdea3259592946](https://polkadot.subscan.io/extrinsic/0xe640a2af11b057a60c509a116c2ffe372c4524493377757c4bbdea3259592946)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**